test/TestImportWithMalformed.py

```diff
@@ -1,10 +1,8 @@
 from collections import namedtuple
 
 import pytest
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.